backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/ValidityDate.java

```diff
@@ -53,37 +57,15 @@ public static ValidityDate create(Column startColumn, Column endColumn) {
 
 	@CheckForNull
 	public CDateRange getValidityDate(int event, Bucket bucket) {
-		// I spent a lot of time trying to create two classes implementing single/multi-column valditiy dates separately.
-		// JsonCreator was not happy, and I could not figure out why. This is probably the most performant implementation that's not two classes.
-
-		if (getColumn() != null) {
-			final Column resolvedColumn = getColumn().resolve();
-			if (bucket.has(event, resolvedColumn)) {
-				return bucket.getAsDateRange(event, resolvedColumn);
-			}
-
-			return null;
+		if (extractor == null){
+			//TODO this is just a workaround: We should actually be using Initializing, which sadly gives us issues with LoadingUtil
+			init();
```

Comment:
Ja das mit der Abhängigkeit hatte ich in den Tests nicht beachtet. Sehe aber auch keine besser Lösung